pkgs/development/ocaml-modules/ocaml-lsp/jsonrpc.nix

```diff
@@ -7,41 +7,47 @@
 , fetchurl
 , lib
 , ocaml
+, version ? "1.16.2"
```

Comment:
I like this solution, but I would like to have @vbgl's thoughts on it

Reply:
This is fine but the default value is wrong: it must be computed from the value of `ocaml.version`.